src/stringify.js

```diff
@@ -266,7 +266,14 @@ var JSON = module.exports;
 
             if (isLong)
             {
-                return value.toString();
+                if (value.unsigned)
+                {
+                    return `{"__longValue":${value.toString()},"__unsigned":true}`;
```

Comment:
is the double underscore something that is common practice? If not, let's remove them and use whatever the common practice is

Reply:
I asked around and it seems there is no double underscore convention. I will change it to single underscores.